flox/core.py

```diff
@@ -1276,6 +1309,8 @@ def _validate_reindex(reindex: bool, func, method, expected_groups) -> bool:
     if method in ["split-reduce", "cohorts"] and reindex is False:
         raise NotImplementedError
 
+    # TODO: Should reindex be a bool-only at this point? Would've been nice but
+    # None's are relied on after this function as well.
```

Comment:
:+1: for bool only , but OK for now. Can you open an issue please?

Reply:
See #155.